docs/README.rst

```diff
@@ -15,7 +15,7 @@ and all its dependencies run
 
 ::
 
-    pip install 'Sphinx >= 1.3' sphinx_rtd_theme
+    pip install -r requirements.txt
```

Comment:
Just like the `Compile the documentation` part of this README, to maintain uniformity shall I mention "run the following command from this dir"?